lib/find-python.js

```diff
@@ -26,7 +26,22 @@ PythonFinder.prototype = {
   win: win,
   pyLauncher: 'py.exe',
   winDefaultLocations: [
-    path.join(process.env.SystemDrive || 'C:', 'Python37', 'python.exe')
+    path.join(process.env.SystemDrive || 'C:', 'Users', process.env.USERNAME || process.env.USER, 'AppData', 'Local', 'Programs', 'Python', 'Python39', 'python.exe'),
+    path.join(process.env.SystemDrive || 'C:', 'Program Files', 'Python39', 'python.exe'),
+    path.join(process.env.SystemDrive || 'C:', 'Users', process.env.USERNAME || process.env.USER, 'AppData', 'Local', 'Programs', 'Python', 'Python39-32', 'python.exe'),
+    path.join(process.env.SystemDrive || 'C:', 'Program Files (x86)', 'Python39', 'python.exe'),
+    path.join(process.env.SystemDrive || 'C:', 'Users', process.env.USERNAME || process.env.USER, 'AppData', 'Local', 'Programs', 'Python', 'Python38', 'python.exe'),
+    path.join(process.env.SystemDrive || 'C:', 'Program Files', 'Python38', 'python.exe'),
+    path.join(process.env.SystemDrive || 'C:', 'Users', process.env.USERNAME || process.env.USER, 'AppData', 'Local', 'Programs', 'Python', 'Python38-32', 'python.exe'),
+    path.join(process.env.SystemDrive || 'C:', 'Program Files (x86)', 'Python38', 'python.exe'),
+    path.join(process.env.SystemDrive || 'C:', 'Users', process.env.USERNAME || process.env.USER, 'AppData', 'Local', 'Programs', 'Python', 'Python37', 'python.exe'),
+    path.join(process.env.SystemDrive || 'C:', 'Program Files', 'Python37', 'python.exe'),
+    path.join(process.env.SystemDrive || 'C:', 'Users', process.env.USERNAME || process.env.USER, 'AppData', 'Local', 'Programs', 'Python', 'Python37-32', 'python.exe'),
+    path.join(process.env.SystemDrive || 'C:', 'Program Files (x86)', 'Python37', 'python.exe'),
+    path.join(process.env.SystemDrive || 'C:', 'Users', process.env.USERNAME || process.env.USER, 'AppData', 'Local', 'Programs', 'Python', 'Python36', 'python.exe'),
+    path.join(process.env.SystemDrive || 'C:', 'Program Files', 'Python36', 'python.exe'),
+    path.join(process.env.SystemDrive || 'C:', 'Users', process.env.USERNAME || process.env.USER, 'AppData', 'Local', 'Programs', 'Python', 'Python36-32', 'python.exe'),
+    path.join(process.env.SystemDrive || 'C:', 'Program Files (x86)', 'Python36', 'python.exe')
```

Comment:
This seems like a lot of boilerplate code with tons of possibilities for typos.  Would there be a way to use a list-driven approach to remove the repetitive code.  `for python in ["Python39", "Python38", "Python37", "Python36"]:`

Reply:
I just wanted to comment the same thing, but with `'39', '38', '37', '36'` :-)

Reply:
@cclauss @mfonville, sure. It should be as simple as constructing the array programmatically somewhere else in the file, assigning that to a variable, then assigning the exported array from the variable. (Instead of directly exporting all this verbose/hard-coded stuff.)